src/xdp-app-info.c

```diff
@@ -155,6 +168,18 @@ xdp_app_info_get_instance (XdpAppInfo *app_info)
   return priv->instance;
 }
 
+const char *
+xdp_app_info_get_desktop_file (XdpAppInfo *app_info)
```

Comment:
Would it be better to call it `xdp_app_info_get_desktop_file_id ()` to be consistent? Same in other locations where you use `desktop_file`

Reply:
Yup, good call